.dockerignore

```diff
@@ -6,3 +6,4 @@
 !go.mod
 !go.sum
 !index.html
+.go
```

Comment:
This doesn't do anything?

Reply:
`$(pwd)/.go/` is my `$GOPATH`. I wish there was a global `.dockerignore`.
I can remove it from from the PR.

Reply:
No problem leaving it, just leave a comment so the next person doesn't ask the same thing or even removes it thinking it doesn't do anything

Reply:
Left with a comment.